<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculatorClass</class>
 <widget class="QMainWindow" name="CalculatorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="inputLine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="RigButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FiveButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RecButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>1/x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DivButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>340</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SqrtButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TwoButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>340</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MulButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>220</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="expression">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="LefButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SevenButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SubButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>280</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FourButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SixButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ZeroButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>400</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BotButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>+-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NineButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EqlButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>400</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DecButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>400</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ThreeButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>340</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CEButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SquareButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>square</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EightButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ACButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>c</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OneButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>409</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Calculator.qrc"/>
 </resources>
 <connections/>
</ui>
